image/mediatypes.go

```diff
@@ -0,0 +1,13 @@
+package image
+
+// mediatype definitions for image components handled in containerd.
+//
+// oci components are generally referenced directly, although we may centralize
+// here for clarity.
+const (
+	MediaTypeDockerSchema2Layer        = "application/vnd.docker.image.rootfs.diff.tar"
```

Comment:
I think it'd be fair to merge it as-is and open an issue for follow-up so we can move forward.